### PR TITLE
Add debug support taskrun

### PR DESCRIPTION
**Pull request by sudhirverma**
Fix: #627

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -987,4 +1046,4 @@
     "validator": "^11.0.0",
     "vscode-kubernetes-tools-api": "1.3.0"
   }
-}
+}
```
Comment (evidolob) [inline package.json:1049]: Add new line, please

**Comment by evidolob** [comment]
@sudhirverma I got this error:
Screenshot 2021-09-27 at 15 18 38
When try to start task run with debug.

And other suggestion, I think it would be better to swap tektonhub and debug view, to keep tektonhub last.

File: src/debugger/debug-tree-view.ts [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  namespace?: string;
+}
+
+export const sessions: Map<string, DebugSessionEntry> = new Map();
+
+
+export async function debugTreeView(): Promise<TektonNode[]> {
+  if (sessions && sessions.size !== 0) {
+    const children = [];
+    for (const [key, value] of sessions) {
```
Comment (evidolob) [inline src/debugger/debug-tree-view.ts:37]: Please fix `'value' is assigned a value but never used`

File: test/debugger/debugExplorer.test.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
+/*-----------------------------------------------------------------------------------------------
+ *  Copyright (c) Red Hat, Inc. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE file in the project root for license information.
+ *-----------------------------------------------------------------------------------------------*/
+
+import * as chai from 'chai';
+import * as sinonChai from 'sinon-chai';
+import sinon = require('sinon');
+import { DebugExplorer } from '../../src/debugger/debugExplorer';
+import { TestItem } from '../tekton/testTektonitem';
```
Comment (evidolob) [inline test/debugger/debugExplorer.test.ts:10]: Please fix `'TestItem' is defined but never used`

File: test/debugger/debugExplorer.test.ts [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+import { TknImpl } from '../../src/tkn';
```
Comment (evidolob) [inline test/debugger/debugExplorer.test.ts:11]: And this `'TknImpl' is defined but never used`

**Review by evidolob — COMMENTED** [review]
Overall works great

File: src/cli-command.ts [hunk 5]
```diff
@@ -408,6 +416,42 @@ export class Command {
     }
   }
 
+  static loginToContainer(container: string, podName: string, namespace: string): CliCommand {
+    if (ocFallBack.get('ocFallBack')) {
+      return newOcCommand('exec', '-it', '-n', namespace, '-c', container, podName, 'bash');
```
Comment (evidolob) [inline src/cli-command.ts:421]: I receive message
`kubectl exec [POD] [COMMAND] is DEPRECATED and will be removed in a future version. Use kubectl exec [POD] -- [COMMAND] instead`.
Could you verify that, and use new type of the command

Comment (sudhirverma) [reply src/cli-command.ts:421]: okay I will use ```kubectl exec mypod -i -t```

Comment (sudhirverma) [reply src/cli-command.ts]: @evidolob done

**Review by evidolob — APPROVED** [review]

**Pull request merged**